assets/src/components/error-screen/index.js

```diff
@@ -19,27 +19,38 @@ import './style.css';
  *
  * @param {Object} props Component props.
  * @param {Object} props.error Error object containing a message string.
- * @param {string} props.finishLink The link to return to the admin.
+ * @param {Object} props.finishLink The link to return to the admin.
```

Comment:
Accepting an object for a prop named `finishLink` seems like a misnomer. Accepting the link and label as separate props seems more appropriate.

Reply:
I agree. I've broken down the `finishLink` object into 2 separate props in 81e29d8.